cmd/kubectl-testkube/commands/tests/renderer/test_obj.go

```diff
@@ -0,0 +1,45 @@
+package renderer
+
+import (
+	"fmt"
+
+	"github.com/kubeshop/testkube/pkg/api/v1/testkube"
+	"github.com/kubeshop/testkube/pkg/ui"
+)
+
+func TestRenderer(ui *ui.UI, obj interface{}) error {
+	test, ok := obj.(testkube.Test)
+	if !ok {
+		return fmt.Errorf("can't use '%T' as testkube.Test in RenderObj for test", obj)
+	}
+
+	ui.Warn("Name:     ", test.Name)
+	ui.Warn("Namespace:", test.Name)
+	ui.Warn("Created:  ", test.Created.String())
+	ui.Warn("Labels:   ", fmt.Sprintf("%v", test.Labels))
+
+	if test.Content != nil {
+		ui.NL()
+		ui.Info("Content")
+		ui.Warn("Type", test.Content.Type_)
+		if test.Content.Uri != "" {
+			ui.Warn("Uri: ", test.Content.Uri)
+		}
+
+		if test.Content.Repository != nil {
+			ui.Warn("Repository: ")
```

Comment:
I would use loop, instead of 6 same lines

Reply:
@vsukhin do you mean loop here? how?
```go
			ui.Warn("  Uri:      ", test.Content.Repository.Uri)
			ui.Warn("  Branch:   ", test.Content.Repository.Branch)
			ui.Warn("  Path:     ", test.Content.Repository.Path)
			ui.Warn("  Username: ", test.Content.Repository.Username)
			ui.Warn("  Token:    ", test.Content.Repository.Token)
```

Reply:
I meant to have something like
type Data struct {
Header string
Value string
}
output := []Data{Header:"  Uri:      ',  Value:  test.Content.Repository.Uri}
for _, warning := range output{
ui.Warn(warning.Header, warning.Value)
}